core/profile.c

```diff
@@ -1160,6 +1160,11 @@ static int calculate_ccr_po2(struct plot_data *entry, const struct divecomputer
 	}
 }
 
+double gas_density(const struct gas_pressures *pressures)
```

Comment:
If this function is used only in this source file, then it should be made of `static` linkage.

I think there is a typo in the commit message: 'diluent' with one 'l'.

Reply:
Addressed your concerns in an updated version.